packages/ui/Media/tests/Media-specs.js

```diff
@@ -0,0 +1,177 @@
+/* global HTMLMediaElement */
+
+import '@testing-library/jest-dom';
+import {createEvent, fireEvent, render, screen} from '@testing-library/react';
+import {createRef} from 'react';
+
+import {getKeyFromSource, Media} from '../Media';
+
+let mediaLoadStub;
+let mediaPlayStub;
+let mediaPauseStub;
+
+const customMediaEventsMap = {
+	custom : 'onCustom'
+};
+
+describe('Media', () => {
+	beforeEach(() => {
+		mediaLoadStub = jest.spyOn(HTMLMediaElement.prototype, 'load').mockImplementation(() => true);
+		mediaPlayStub = jest.spyOn(HTMLMediaElement.prototype, 'play').mockImplementation(() => true);
+		mediaPauseStub = jest.spyOn(HTMLMediaElement.prototype, 'pause').mockImplementation(() => true);
+	});
+
+	afterEach(() => {
+		mediaLoadStub.mockRestore();
+		mediaPlayStub.mockRestore();
+		mediaPauseStub.mockRestore();
+	});
+	test('should use the same node when changing the `source`', () => {
```

Comment:
Could you add an empty line between each test?

Reply:
Added